app/containers/LeftMenu/MainLinks.jsx

```diff
@@ -156,7 +156,7 @@ const MainLinks = ({ profile, currClientHeight, isGlobalAdmin }) => {
       {!singleCommId && (
         <A1 to={routes.feed()} name="feed" route={route}>
           <IconLg className="mr-2" icon={myFeedIcon} />
-          <FormattedMessage {...messages.myFeed} />
+          {profile ? <FormattedMessage {...messages.feed} /> : <FormattedMessage {...messages.myFeed} />}
```

Comment:
```
<FormattedMessage id={messages[profile ? 'feed' : 'myFeed'].id} />
```

Reply:
fixed: https://github.com/peeranha/peeranha-web/pull/789/commits/42fd8b445ff10f089f4afc944f72b676524bdb93
thanks for this solution)